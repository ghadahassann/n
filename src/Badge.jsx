export default function Badge({ children }) {
  return (
    <span role="status" className="badge">
      {children}
    </span>
  );
}
