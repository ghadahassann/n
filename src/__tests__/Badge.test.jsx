import { render } from '@testing-library/react';
import Badge from '../Badge';

test('matches snapshot (small & stable)', () => {
  const { container } = render(<Badge>New</Badge>);
  expect(container.firstChild).toMatchSnapshot();
});
