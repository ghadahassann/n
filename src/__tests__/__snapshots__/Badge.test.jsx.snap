// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`matches snapshot (small & stable) 1`] = `
<span
  class="badge"
  role="status"
>
  New
</span>
`;
